Store Customization > Homepage Template 1: Review layout and spacing in between patterns

albarin opened the pull request:
<!-- Please do not remove any information from this pull request. Instead, add N/A or leave blank if not applicable -->

## What

This PR adds a group block with padding to the `Product Collection 3 Columns`, `Featured Category Triple` and the `Social: Follow us on social media` patterns.

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11088

## Why

To adapt to the designs shown on https://github.com/woocommerce/woocommerce-blocks/issues/11088

## Testing Instructions

<!-- Write these steps from the perspective of a "user" (merchant) familiar with WooCommerce. No need to spell out the steps for common setup scenarios (eg. "Create a product"), but do be specific about the thing being tested. Include screenshots demonstrating expectations where that will be helpful. -->

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post and insert all the patterns shown in the screenshot below 👇
- `Minimal 5-column products row` corresponds to `Product Collection 5 Columns`
2. Make sure the design is the same and the spacing between patterns matches.

* [ ] Do not include in the Testing Notes <!-- Check this box if this PR can't be tested (ie: it makes changes to tests, coding standards, docblocks, etc.). -->
* [ ] Should be tested by the development team exclusively <!-- Check this box if this PR should be tested by the development team exclusively (ie: it doesn't include user-facing changes or it can't be tested without manually modifying the code). -->

## Screenshots or screencast

<img width="524" alt="Screenshot 2023-10-02 at 17 39 07" src="https://user-images.githubusercontent.com/15730971/272015191-e07193cc-07e2-4a3e-b88e-29f4654850db.png">

## WooCommerce Visibility

<!-- Check this documentation link (../docs/blocks/feature-flags-and-experimental-interfaces.md) to see if the change is visible in WooCommerce core, part of the feature plugin, or experimental. -->
Required:

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental
* [ ] N/A

## Checklist

Required:
* [x] This PR has either a `[type]` label or a `[skip-changelog]` label.
* [ ] This PR is assigned to a milestone.

Conditional:
* [x] This PR has a changelog description (if `[skip-changelog]` label is not present).
* [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR adds/removes an experimental interfaces, and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR has been accessibility tested.
* [ ] This PR has had any necessary documentation added/updated.

## Changelog
<!-- Provide a brief, descriptive summary of the changes in this PR. Include potential impacts on different parts of the product. Example: "Updated the checkout process to streamline the experience for users and reduce the number of steps." -->

> Add group and padding to the `Product Collection 3 Columns`, `Featured Category Triple` and the `Social: Follow us on social media` patterns.

albarin committed:
Add titles to patterns and set the aligment to Wide

albarin committed:
Replace product query patterns with product collection ones

albarin committed:
Add group and spacing to the 3-cols and social patterns

tjcafferkey reviewed (approved):
Looks good other than the issues already described here https://github.com/woocommerce/woocommerce-blocks/pull/11129#issuecomment-1748580992 so approving since we have agreed to fix these in a follow-up PR

tjcafferkey commented:
Just noting some visual inconsistencies here:

* The title for the testimonials pattern appears to not be fully left alignment in the editor only.
![Screenshot 2023-10-05 at 12 20 02](https://github.com/woocommerce/woocommerce-blocks/assets/8639742/f2f7a67f-b4ac-4e5b-9203-76b01409f5a8)

* In both the editor and the frontend there seems to be some alignment issues. Most noticeable in the bottom 3 patterns
![Screenshot 2023-10-05 at 12 21 24](https://github.com/woocommerce/woocommerce-blocks/assets/8639742/d0359bd2-35a7-46bd-a0bf-5a80468a47fd)

* White space between patterns appears smaller compared to the designs.
![Screenshot 2023-10-05 at 12 20 08](https://github.com/woocommerce/woocommerce-blocks/assets/8639742/1af1f4af-a165-4e5a-8aed-842384dc4023)

albarin committed:
Fix alignment issue in the testimonals 3 in the editor

albarin committed:
Add padding to the featured catergory triple pattern

albarin commented:
@tjcafferkey do you want to have another look?

>  The title for the testimonials pattern appears to not be fully left alignment in the editor only.

Good catch! Fixed! 

>  In both the editor and the frontend there seems to be some alignment issues. Most noticeable in the bottom 3 patterns

Yes, I saw that and fixed it in one of the other Homepage PRs, I merged them and resolved conflicts and this is not happening anymore.

>  White space between patterns appears smaller compared to the designs.

Fixed by adding a bit of padding on the `Featured Category Triple`.

tjcafferkey commented:
Thanks for addressing them @albarin there is one final alignment issue with the triple category pattern but I think we’re safe to merge and we can address that pattern in a separate PR since it spans multiple templates not just this one.
